### DISCUSSION TITLE
表格识别 报错 IndexError: list index out of range

### DISCUSSION THREAD

**Discussion by reevexia** [PaddlePaddle/PaddleOCR#13993] [Q&A]
![90f03c0dbd9123ba2ce8e4c154cde47](https://github.com/user-attachments/assets/d927d872-47a4-442f-af8e-5f3b921c311a)

**Comment by GreatV** [comment]
麻烦提供一下运行环境和最小可复现的demo

**Comment by reevexia** [comment]
**PaddleOCR/deploy/hubserving/structure_table/params.py**

# Copyright (c) 2022 PaddlePaddle Authors. All Rights Reserved.
#
# Licensed under the Apache License, Version 2.0 (the "License");
# you may not use this file except in compliance with the License.
# You may obtain a copy of the License at
#
#     http://www.apache.org/licenses/LICENSE-2.0
#
# Unless required by applicable law or agreed to in writing, software
# distributed under the License is distributed on an "AS IS" BASIS,
# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
# See the License for the specific language governing permissions and
# limitations under the License.

from __future__ import absolute_import
from __future__ import division
from __future__ import print_function

from deploy.hubserving.ocr_system.params import read_params as pp_ocr_read_params


def read_params():
    cfg = pp_ocr_read_params()

    # params for table structure model
    cfg.table_max_len = 488
    cfg.table_model_dir = "./inference/ch_ppstructure_mobile_v2.0_SLANet_infer/"
    cfg.table_char_dict_path = "./ppocr/utils/dict/table_structure_dict.txt"
    cfg.show_log = False
    return cfg

PaddleOCR 代码为gitee上最新代码
运行环境为  docker 镜像 2.6.2-gpu-cuda11.2-cudnn8.2-trt8.0 版本

#使用hubserving启动的服务
#表格识别
hub install deploy/hubserving/structure_table
#运行表格识别服务
export CUDA_VISIBLE_DEVICES=0
nohup hub serving start -c deploy/hubserving/structure_table/config.json > ppocr_table.log 2>&1 &

**Reply by reevexia** [reply]
![table](https://github.com/user-attachments/assets/cacaecf6-95fb-487d-b08b-dd0d8fa21469)
待识别的图片

**Comment by Liyulingyue** [comment]
可以参考一下 https://github.com/PaddlePaddle/PaddleX/blob/release/3.0-beta1/docs/pipeline_deploy/service_deploy.md
